app/components/ScrollToButton.tsx

```diff
@@ -21,20 +20,9 @@ export function useScrollToEvent(scheduleIndex) {
   const currentEventIndex = useSharedValue(-1)
   const currentlyViewingEvents = useSharedValue<number[]>([])
   const currentlyViewingSchedule = useSharedValue(0)
-  const isFocused = useSharedValue(false)
-
-  useFocusEffect(
-    useCallback(() => {
-      isFocused.value = true
-      return () => {
-        isFocused.value = false
-      }
-    }, []),
-  )
```

Comment:
This seems it's not necessary anymore.  The issue that made me add this in the previous PR doesn't seem to exist anymore.

Reply:
Spent some time to figure out what fixed this but could find a reason why the [issue](https://github.com/infinitered/ChainReactApp2023/pull/72) disappeared. 🤷🏼‍♀️